### ISSUE TITLE
Cannot select Flight Simulator 2020 (Microsoft Store Edition)

### ISSUE THREAD

**Issue by PadinnPlays** [ViGEm/HidHide#53]
Not sure if this is a bug with the Windows UMP apps but I cannot select Flight Simulator 2020.exe. It will give an error saying I don't have permissions. Tried the new beta Xbox app, which lets you relocate the folder outside the WindowsApp protected one, but the issue still occurs. Open to suggestions!

**Comment by nefarius** [comment]
UWP applications are technically not individual processes in the classic sense. It's a sandboxed overly complicated mess. I'm afraid this type of game is not affectable by HidHide.

**Comment by PlaceboPRS** [comment]
Hope it's ok to bump an old issue. I was having the same problem with Fallout 76, what I did was copy a random exe to the FO76 folder, renamed the game's exe (Project76_GamePass.exe) to Project76_GamePass.bak, renamed my random exe to Project76_GamePass.exe then added it via the Hidhide interface. Then deleted the random exe and returned Project76_GamePass.bak to the correct name. And then when I ran FO76 it correctly detected my DS4 and worked with it despite hidhide being enabled :)

Yes a convoluted method but it worked! :)
